torch/testing/_internal/common_methods_invocations.py

```diff
@@ -358,6 +358,20 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_tensor_hsplit_vsplit_dsplit(op_info, device, dtype, requires_grad):
+    return (SampleInput(make_tensor((S, S, S), device, dtype,
```

Comment:
A few notes:
- add a test with an actual Python scalar
- first and third SampleInputs are redundant
- split sample inputs for hsplit, vsplit, and dsplit and test 0D, 1D and 3D cases for hsplit, 2D and 3D for vsplit, and 3D for hsplit (like this currently does but also with the Python scalar arg case)